test/factories/jobs.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  factory :job do
+    title Faker::Lorem.characters(7)
+    source_url Faker::Internet.url
+    source Faker::Lorem.characters(7)
+    city Faker::Address.city
+    state Faker::Address.state
+    country Faker::Address.country
+    description Faker::Lorem.paragraph
```

Comment:
All the attributes being set above this comment should get assigned with block to evaluate the Faker data lazily each time the factory is used instead of at test load time. Otherwise all jobs created with this factory will have the same values.

e.g. `title { Faker::Lorem.characters(7) }`